man/systemd.network.xml

```diff
@@ -1396,6 +1396,15 @@
           </listitem>
         </varlistentry>
 
+        <varlistentry>
+          <term><varname>MaxClientAttempt=</varname></term>
+          <listitem>
+            <para>Specifies how many attempts the DHCPv4 client initiate reconfiguration. Takes a number or
+            <literal>infinity</literal>. Defaults to 6.
+            </para>
```

Comment:
Let's call this "MaxAttempts=". It certainly should be plural. And "Client" is unneeded, the whole [DHCP] section is about client settings. I think it'd also be nice to add a note about exponential backoff:

"""
Specifies how many times the DHCPv4 client configuration should be attempted. Takes a number or "infinity". Defaults to 6.

The time between retries is increased exponentially, so the network will not be overloaded even if this number if high.
""""

Reply:
It's actually defaults to ```infinity```. So updated accordingly.

Reply:
> It's actually defaults to `infinity`. So updated accordingly.

Yeah. Thank you.